### PR TITLE
Styling (second part)

### LINKED ISSUE #13: 13. Como usuario, quiero que la aplicación tenga un aspecto profesional y acogedor.
Las últimas 2 semanas del proyecto funcionarán de manera un poco diferente al resto. El equipo **se autoorganizará** en torno a un **proceso de diseño ligero** para crear una apariencia elegante para tu aplicación.

Esto requerirá que **tomen decisiones juntos como un equipo completo** y luego implementen su visión. El proceso que se describe aquí es nuestra mejor sugerencia sobre cómo proceder, pero **¡eres libre de tomar el enfoque que consideres adecuado!**

¡Notarás que este es el último problema en el tablero! Esto se debe a que la expectativa es que **escriban en equipo las tareas necesarias para completar la implementación** de su dirección de diseño.

Como siempre, **apóyate en tus mentores** para que te orienten si no estás seguro de cómo proceder.

## Proceso de diseño

1. ¿Cuál es el tono que buscas? ¿Qué sentimiento estás intentando transmitir? Piense de 3 a 5 palabras que capturen mejor cómo desea que los usuarios perciban su aplicación.
    - Ejemplos:
        - Gracioso
        - Limpio
        - irónico
        - profesional
        - [Muchos más ejemplos de palabras] (http://static1.squarespace.com/static/5723ff8086db4345e722e7be/573248885f21e0a9e45338ef/57b6a206579fb39bc0e08e2a/1471586826621//img.jpg)
2. Cree un [estado de ánimo]  (https://milanote.com/guide/moodboard) [tablero] (https://www.canva.com/learn/make-a-mood-board/).     Un tablero de estado de ánimo ayuda a todos a estar en la misma página en términos de la dirección en la que desea tomar su diseño. Agregue elementos a su cuadro de estado de ánimo que representen el tono que está tratando de lograr (Paletas de colores).
3. Decida los elementos fundamentales de su diseño, incluidos los siguientes:
    - **Tipografía** ← [Google Fonts] (https://fonts.google.com/) es una buena opción para encontrar fuentes web que puede utilizar en su aplicación. Tienen cientos de tipos de letra para elegir, ¡pero considere la usabilidad en la que elija!
    - **Colores** ← ¡Asegúrate de que sean [accesibles] (https://accessible-colors.com/)!
    - **Iconografía / Imágenes** ← ¡Empiece aquí para [encontrar iconos gratuitos que pueda usar en su aplicación] (https://www.canva.com/learn/free-icons-download/)!
4. Decida si desea codificar su CSS desde cero o integrar un marco CSS. ¡Para cada uno hay pros y contras!
    - Vanilla CSS te da más control, pero puede ser más difícil y / o tardar más en lograr cierto efecto.
    - Los marcos lo ayudan a hacer las cosas más rápido, pero a menudo a costa de no brindarle un control total del diseño.
5. Si opta por utilizar un marco, evalúe sus opciones para asegurarse de elegir uno que se adapte a sus necesidades.
    - Considera lo siguiente:
        - Calidad de la documentación
        - Facilidad de integración
        - Construya teniendo en cuenta la accesibilidad
        - Actividad comunitaria
    - Los marcos de calidad que podría considerar incluyen los siguientes:
        - [Bootstrap] (https://getbootstrap.com/)
        - [Interfaz de usuario de Chakra] (https://chakra-ui.com/)
        - [IU de material] (https://material-ui.com/)
        - [IU semántica] (https://semantic-ui.com/)
        - [TailwindCSS] (https://tailwindcss.com/)

### LINKED ISSUE #30: 16. Aplicar estilos a la vista para adicionar un item a la lista de compras
En la aplicación dentro de la vista de "Adicionar un item" (Add Item), se necesita añadir estilos CSS, para mejorar la presentación y diseño de la vista.

AC:

- Crear una caja que contenga el formulario
- Cambiar los radio button a botones
- Estilizar los botones similares a las vistas anteriores, como ser: página principal.

Referencia de diseño:

Add Item v1

### LINKED ISSUE #31: 17. Aplicar estilos a la vista para mostrar la lista de compras
En la aplicación dentro de la vista de "Listado de Compras" (List View), se necesita añadir estilos CSS, para mejorar la presentación y diseño de la vista.

AC:

- Organizar los items de la lista en forma de cajas
- Cambiar los botones por iconos
- Aplicar colores para los diferentes estimados de tiempos para la siguiente compra (nextPurchased), como ser: soon, kind of soon, not soon, kind of soon, también cuando este en estado inactivo.

Referencia de diseño:

List-v4

### PR DESCRIPTION
**Pull request by Lukaina**
## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |

## Related Issue

- Closes #13 
- Closes #30 
- Closes #31

## Description

As a user, I want the app to look professional and welcoming. In the last 2 weeks of the project the team self-organized around a light design process to create a sleek look and feel for the application. We wanted it to have a minimalist style and under the psychology of color, we chose red and yellow in the application, as they are colors widely used for example in restaurants and any food business. This is because these colors affect our appreciation of taste and, in this way, influence our appetite.

The first week we worked on the main view and the empty list view, this week, we focus and finish with the Add Item and List view.

### Issue #30 
In the application within the Add Item view, CSS styles were added to improve the presentation and design of the view.

### Issue #31 
In the application within the List View, CSS styles were added to improve the presentation and design of the view.

### Issue #30 

- Create a box containing the form.
- Change radio buttons to buttons.
- Stylize the buttons similar to the previous views, such as: home page.

### Issue #31

- Organize list items in the form of boxes.
- Change buttons to icons.
- Apply colors for the different time estimates for the next purchase (nextPurchased), such as: soon, kind of soon, not soon, also when in the inactive state.

### List View
ListView

### Add Item View
AddItem1
AddItem2

### CODE REVIEW

File: src/components/FormProduct/index.jsx [hunk 1]
```diff
@@ -14,88 +14,139 @@ export const FormProduct = () => {
...
+            nextPurchase: Number(nextPurchase),
+            creationDate: new Date(),
+          };
+          addProduct(editedProduct);
+          setProduct({ ...initialStateProduct });
+          setError(false);
+          setErrorMsg('');
+          setSuccessMsg(true);
+        }
+      } else {
```
Comment (indiragp) [inline src/components/FormProduct/index.jsx:63]: I find it odd that you are using a single message. 
If the app is missing both the name and next purchase date, you want both messages to show at the same time. 
You don't want the user to click 'Add Item' for every single field to notice that it is required. 
No input -> submit -> Error: You are missing the name
User adds the name -> submit -> Error: You are missing next purchase date
User adds the purchase date -> submit -> success

Ideally:
No input -> Submit -> Error: You are missing name (next to name), You are missing next purchase date 
User adds both, the name and next purchase date -> Success!

File: src/components/FormProduct/index.jsx [hunk 2]
```diff
@@ -14,88 +14,139 @@ export const FormProduct = () => {
...
+        setError(true);
+        setErrorMsg('Next Purchase is required');
```
Comment (indiragp) [inline src/components/FormProduct/index.jsx]: Next Purchase *date*

File: src/components/FormProduct/index.jsx [hunk 3]
```diff
@@ -14,88 +14,139 @@ export const FormProduct = () => {
...
+              onChange={handleRadioNextPurchase}
+              type="radio"
+              required="required"
+              name="options"
+              value={6}
+              id="option-soon"
+              checked={product.nextPurchase === '6' ? true : false}
+            />
+            <label className="btn btn-soon" htmlFor="option-soon">
+              <i class="bi bi-stopwatch-fill" style={{ fontSize: 24 }}></i>
```
Comment (indiragp) [inline src/components/FormProduct/index.jsx]: I am not sure of how valuable the selection of the icons is, they don't seem to communicate the desired message, but I look forward to hearing more about the process selection during our weekly sync

**Review by indiragp — APPROVED** [review]
Great job team! 
I just left a couple of comments for you to consider.

**Review by gmzjuliana — APPROVED** [review]

File: src/components/FormProduct/index.jsx (hunk 1 above)
Comment (alxmcr) [reply src/components/FormProduct/index.jsx:63]: Thanks Indira! I appreciate your comment and observation. Yes, it's a better solution. Fixed.

image
image
image
image

File: src/components/FormProduct/index.jsx (hunk 2 above)
Comment (alxmcr) [reply src/components/FormProduct/index.jsx]: Thanks for your observation. Fixed!

image

File: src/components/FormProduct/index.jsx (hunk 3 above)
Comment (alxmcr) [reply src/components/FormProduct/index.jsx]: Thanks! OK, in our weekly sync we can talk about it.

**Pull request merged**